README.md

```diff
@@ -24,10 +24,10 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: cli/gh-extension-precompile@v1
+      - uses: actions/checkout@v4
+      - uses: cli/gh-extension-precompile@v2
         with:
-          go_version: "1.16"
+          go_version_file: go.mod
```

Comment:
I decided to change all examples to use `go_version_file` instead of `go_version`, including this main usage example. Let me know if you disagree 😁